fiftyone/utils/eval/detection.py

```diff
@@ -183,6 +183,14 @@ def __init__(
         self.iou = iou
         self.classwise = classwise
 
+    @property
+    def additional_fields_required(self):
```

Comment:
style nit: we use active tense for method/property names. So prefer `requires_additional_fields` to `additional_fields_required`